styles/main.header.scss

```diff
@@ -24,11 +23,26 @@
 @import "history";
 @import "historyTable";
 
+// Define some utility classes
```

Comment:
Add some utility classes. This removes some inline classes and eases basic formatting on the history table.